src/inspector_socket.cc

```diff
@@ -1,8 +1,6 @@
 #include "inspector_socket.h"
 
-#ifdef NODE_EXPERIMENTAL_HTTP_DEFAULT
 #define NODE_EXPERIMENTAL_HTTP
```

Comment:
Where is this value `NODE_EXPERIMENTAL_HTTP` consumed?
If it's required, should we rename it as `llhttp` won't be experimental anymore after this commit is pushed?

Reply:
It’s consumed in `http_parser_adaptor.h`.

I agree that the original name was not a great choice, but I am not sure that it would be worth the churn of renaming – if you feel strongly about it, you can feel free to push a new commit to this PR or open a new one afterwards?

Reply:
Sure!